debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/SnapshotReader.java

```diff
@@ -532,10 +532,15 @@ protected SourceRecord replaceOffset(SourceRecord record) {
     }
 
     protected void enqueueSchemaChanges(String dbName, String ddlStatements) {
-        if (context.includeSchemaChangeRecords() &&
-                context.makeRecord().schemaChanges(dbName, ddlStatements, super::enqueueRecord) > 0) {
-            logger.debug("Recorded DDL statements for database '{}': {}", dbName, ddlStatements);
+        if (!context.includeSchemaChangeRecords() || ddlStatements.length() == 0) {
+            return;
         }
+        logger.info("Recorded the following DDL statements for database '{}':", dbName);
+        Arrays.stream(ddlStatements.split(";" + System.lineSeparator())).forEach(ddlStatement -> {
```

Comment:
Oh, sorry for that, I did a duplicated thing https://github.com/DataPipelineInc/debezium/commit/3b9dddb51fd461d4e5b63fd5453d68cecf92cf91